### PR TITLE
📊  state_capacity: improve metadata mgm

### PR DESCRIPTION
**Pull request by lucasrodes**
Improves metadata on state_capacity indicators:

- Add topic tags
- Better binary (flag) indicator desscription
- Add key information

### CODE REVIEW

File: etl/steps/data/garden/countries/2023-09-25/isd.meta.yml [hunk 1]
```diff
@@ -17,51 +19,39 @@ dataset:
...
+      description_key:
+        - |-
+          Butcher and Griffiths consider a country a sovereign state if it has a population of at least 10,000 people, has internal control over much of its territory, and is externally recognized by its peers and controls its foreign relations.
+      display:
+        numDecimalPlaces: 0
+
     variables:
       number_countries:
         title: Number of states in a region
         description_short: |-
```
Comment (veronikasamborska1994) [inline etl/steps/data/garden/countries/2023-09-25/isd.meta.yml:46]: Number of sovereign states in a region, defined as entities with a minimum population of 10,000, substantial internal territorial control, and external peer recognition with foreign relations autonomy.

File: etl/steps/data/garden/countries/2023-09-29/cow_ssm.meta.yml [hunk 2]
```diff
@@ -23,31 +38,9 @@ tables:
         description_short: |-
           Number of states listed in the state system membership list for a given region.
```
Comment (veronikasamborska1994) [inline etl/steps/data/garden/countries/2023-09-29/cow_ssm.meta.yml:39]: also possibly a more informative description_short here - e.g. `The number of states in a region's state system membership list depends on changing sovereign state criteria: before 1919, a population above 500,000 and recognition by France and the UK were required; after 1919, either League of Nations or UN membership, or a population over 500,000 recognized by two major powers, was necessary.`

File: etl/steps/data/garden/countries/2023-09-25/gleditsch.meta.yml [hunk 3]
```diff
@@ -24,31 +38,9 @@ tables:
         description_short: |-
           Number of states (and microstates) in each region.
```
Comment (veronikasamborska1994) [inline etl/steps/data/garden/countries/2023-09-25/gleditsch.meta.yml:39]: again maybe something more informative for the description_short here? `Number of states and microstates in each region reflects their status as sovereign states, defined by their relatively autonomous administration of territory and recognition of independence by local actors and other states.`

File: etl/steps/data/grapher/countries/2023-09-29/cow_ssm.meta.yml [hunk 4]
```diff
@@ -3,5 +3,5 @@ tables:
     variables:
       is_present:
         title: Sovereign state
-        description_short: Whether the country is present in the COW SSM dataset.
+        description_short: The country is identified as a sovereign state by Correlates of War - State System Membership dataset.
```
Comment (veronikasamborska1994) [inline etl/steps/data/grapher/countries/2023-09-29/cow_ssm.meta.yml:6]: honestly feel free to ignore ofc but again I think it'd be good to note something about the classification criteria of this source in the subtitle - e.g., `Under the Correlates of War criteria, pre-1919 sovereign state status required over 500,000 people and Franco-British recognition; post-1919, it needed League of Nations or UN membership, or similar population size with recognition from two major powers.`

File: etl/steps/data/grapher/countries/2023-10-01/gleditsch.meta.yml [hunk 5]
```diff
@@ -3,5 +3,5 @@ tables:
     variables:
       is_present:
         title: Sovereign state
-        description_short: Whether the country is present in the Gleditsch and Ward countries dataset.
+        description_short: The country is identified as a sovereign state by Gleditsch and Ward dataset.
```
Comment (veronikasamborska1994) [inline etl/steps/data/grapher/countries/2023-10-01/gleditsch.meta.yml:6]: same here - e.g.,  `The identification of a country as a sovereign state by Gleditsch and Ward depends on its autonomous territorial administration and recognition of independence by both local actors and other states.`

File: etl/steps/data/grapher/countries/2023-10-01/isd.meta.yml [hunk 6]
```diff
@@ -3,5 +3,5 @@ tables:
     variables:
       is_present:
         title: Sovereign state
-        description_short: Whether the country is present in the ISD countries dataset.
+        description_short: The country is identified as a sovereign state by the ISD dataset.
```
Comment (veronikasamborska1994) [inline etl/steps/data/grapher/countries/2023-10-01/isd.meta.yml:6]: again here - e.g., `A country is recognized as a sovereign state by the ISD if it has a minimum population of 10,000, significant internal control over its territory, and is externally acknowledged by peers with autonomy in its foreign relations.`

**Review by veronikasamborska1994 — APPROVED** [review]
looks good! mainly subjective preferences on what I think would be clearer subtitles but you might disagree!

File: etl/steps/data/garden/countries/2023-09-25/isd.meta.yml (hunk 1 above)
Comment (lucasrodes) [reply etl/steps/data/garden/countries/2023-09-25/isd.meta.yml:46]: Leaving as-is, as Bastian proposed this definition.

File: etl/steps/data/garden/countries/2023-09-29/cow_ssm.meta.yml (hunk 2 above)
Comment (lucasrodes) [reply etl/steps/data/garden/countries/2023-09-29/cow_ssm.meta.yml:39]: this is already captured in `definitions.common.description_key`

File: etl/steps/data/garden/countries/2023-09-25/gleditsch.meta.yml (hunk 3 above)
Comment (lucasrodes) [reply etl/steps/data/garden/countries/2023-09-25/gleditsch.meta.yml:39]: this is already captured in `definitions.common.description_key`

File: etl/steps/data/grapher/countries/2023-09-29/cow_ssm.meta.yml (hunk 4 above)
Comment (lucasrodes) [reply etl/steps/data/grapher/countries/2023-09-29/cow_ssm.meta.yml:6]: this is already captured in `definitions.common.description_key`

File: etl/steps/data/grapher/countries/2023-10-01/isd.meta.yml (hunk 6 above)
Comment (lucasrodes) [reply etl/steps/data/grapher/countries/2023-10-01/isd.meta.yml:6]: Leaving as-is, as Bastian proposed this definition.

**Comment by lucasrodes** [comment]
Thanks for carefully looking at this, @veronikasamborska1994

Most of the suggestions are captured by `definitions.common.decription_key`.

**Pull request merged**